frontend/assets/stylesheets/components/_action.scss

```diff
@@ -97,6 +97,29 @@
     cursor: default;
 }
 
+.action--snap {
```

Comment:
Seems to be a lot of magic numbers in this rule, are they in any way related to our spacing vars?